### ISSUE TITLE
Feature Request: Add columns to display recently closed issues

### ISSUE THREAD

**Issue by pllim** [sosey/repo-summary#17]
This was requested by @hcferguson:

*It would be great to have a column that shows the number of issues closed in the past week and another for the past month. *

**Comment by hcferguson** [comment]
I was wondering where this repo was....

While useful to have these columns on the HTML page, it would also be useful to just have a script to grab the time series of issue openings and closings for all our repos and save them in tables somewhere for making plots.

**Comment by pllim** [comment]
Maybe related to #7 ? (I have been meaning to write such a script but haven't found time...)

**Comment by sosey** [comment]
@hcferguson yah, that's a different script, or function that could be added. This  repo is just a library of functions....

example, this will return a json payload that has all issues, that have been open for a long time, in the asdf repository, sorted in descending order.

```
https://api.github.com/repos/spacetelescope/asdf/issues?state\=open&sort\=long-running\&direction\=desc&filter=all
```

You can also get all the statistics that git keeps for each repo, that's a larger call, but you can do whatever you like with the data.  #2 was a suggestion to make a repository of html files, where subpages for each repo could contain more detailed information. 

I wanted to make sure this was working reasonably, and wanted, before throwing much more effort into it... :wink:   before it gets more complex I need to get the OAuth working correctly so that we aren't limited to 60querries an hour...

**Comment by sosey** [comment]
4d150a2a723fc7dcd801658c2fa186fe369dfd9a solves some of this.  There's a closed issues columns, just need to make it in the last x timeframe.

I'm saving all the data for all repos though ... I'll post an example of the different things you can do with this library soon

**Comment by sosey** [comment]
closed issues by week and month added in 99efeed67db26b84eb99a6cfabc142ace1a6b54e
